browser/src/control/Control.PartsPreview.js

```diff
@@ -169,6 +171,7 @@ L.Control.PartsPreview = L.Control.extend({
 
 		var bottomBound = this._getBottomBound();
 		for (var prev = 0; prev < this._previewTiles.length; prev++) {
+			L.DomEvent.on(this._previewTiles[prev], 'contextmenu', function(e) { this._contextMenuHandler(e, this._previewTiles[prev]); } , this);
```

Comment:
this adds a new handler every time to every slide.

Reply:
We have to add intentionally else `_contextMenuHandler` event not triggered when we switch to master view mode and Since In master view and Normal View Image ID is same so by default It will open context menu.

Reply:
Done this one.